package.json

```diff
@@ -8,7 +8,7 @@
     "performances"
   ],
   "scripts": {
-    "postinstall": "scripts/cli init_submodule",
+    "postinstall": "scripts/cli init_submodule && yarn workspace @datadog/browser-worker build",
```

Comment:
Or, like you mentioned on slack with me, we could use a placeholder and replace it with the worker string like we do with the build-env.js script. I've no strong opinion about it, so let see what the other have to say 🙂

Reply:
For the record: we discussed about using an approach similar to `build-env` to inject the worker string in packages/bundles. To do that, we would need:

* a way to synchronously build the package, as building the webpack configuration is currently synchronous and thus require a synchronous `build-env` definition (see [webpack config](https://github.com/DataDog/browser-sdk/blob/53c211aedbaefa990adf9339965748d09e5fef01/webpack.base.js#L67-L73))

* a separate build script from `build-env` (or at least some extra conditions), since `build-env` is used to build the worker, it cannot also provide the worker string

It is definitely doable, but a bit more complex, so we'll just move forward with the current approach, and reassess if it's too cumbersome.